src/main.ts

```diff
@@ -13,6 +16,10 @@ const start = async () => {
 	app.listen(port, () => {
 		getLogger("frontend-app").info(`started at port ${port}`);
 	});
+	const lag = createLag(1000);
+	setInterval(() => {
+		statsd.histogram(metricLag.lagHist, lag(), { }, { });
```

Comment:
Do we have any attributes to identify this 1 sec delay? Maybe pass some extra params just in case.

Reply:
Sorry @krazziekay not sure I understand... 1 sec is just a frequency how often we want to send this metric to sfx...

Reply:
Approved then, its just for testing anyways right?

Reply:
Yeah, it doesn't affect prod. Just to make sure node.js' event loop is not lagging behind (or to identify that this is the problem...)